### ISSUE TITLE
I am facing issue in height adjustment in concave effect 

### ISSUE THREAD

**Issue by amitwd** [lukeed/bee3d#83]
I am using Bee3d on my website https://www.theartzhub.com/art/index.php it is not looking right in mobile view whenever I try to fix height the page having started to scroll, please check and give me a suggestion how to make it right

**Comment by lukeed** [comment]
Hello, did you fix this since writing your message? I don't the problem on my device

**Comment by amitwd** [comment]
yes i tried to fix and its fixed now but with an external div with overflow
hidded
